WIP-CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 ### Changed
 - The Kubernetes status Grafana dashboard (new node filter, new graphs for CPU/Memory requests and limits per node, updated graphs for CPU/Memory usage/requests)
+- bash scripts are now migrated to yq-v4.26.1
```

Comment:
I think you should add this as a release note. And also point out that you need to have yq version 4 as an alias `yq4` if you're not using the `get-requirements.yaml` playbook